test/e2e/wmco_test.go

```diff
@@ -49,7 +49,11 @@ func TestWMCO(t *testing.T) {
 	t.Run("create", creationTestSuite)
 	t.Run("network", testNetwork)
 	t.Run("upgrade", upgradeTestSuite)
-	t.Run("reconfigure", reconfigurationTest)
+	// The reconfigurationTestSuite must be run directly before the deletionTestSuite. This is because we do not
+	// currently wait for nodes to fully reconcile after changing the private key back to the valid key. Any tests
+	// added/moved in between these two suites may fail.
+	// This limitation will be removed with https://issues.redhat.com/browse/WINC-655
```

Comment:
This also needs to be mentioned in `./hack/run-ci-e2e-test.sh` which is what actually kicks  off the test.

Reply:
Sure, I will update.